src/components/Chat/InputMessage.js

```diff
@@ -139,10 +139,10 @@ const styles = StyleSheet.create({
   },
   input: {
     flex: 1,
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: colors.lightgrey
   },
   btn: (disable) => ({
-    backgroundColor: disable ? COLORS.gray9 : COLORS.holyTosca,
+    backgroundColor: disable ? colors.gray1 : colors.bondi_blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been changed. Ensure that the new colors (`colors.lightgrey`, `colors.gray1`, and `colors.bondi_blue`) are equivalent to the old ones (`COLORS.lightgrey`, `COLORS.gray9`, and `COLORS.holyTosca`) in terms of user experience and accessibility.
```diff
- backgroundColor: COLORS.lightgrey
+ backgroundColor: colors.lightgrey
- backgroundColor: disable ? COLORS.gray9 : COLORS.holyTosca,
+ backgroundColor: disable ? colors.gray1 : colors.bondi_blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->